tutorials/3d/standard_material_3d.rst

```diff
@@ -734,12 +734,17 @@ Material Settings
 Render priority
 ---------------
 
-The rendering order of objects can be changed, although this is mostly
+This is mostly
```

Comment:
Why this change? In either case this new sentence is not very clear, the "this" part is confusing IMO when nothing is referred to

Reply:
Ah. It was a run on sentence, originally I'd put the other one first. Will fix.